Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -333,10 +333,21 @@ def bottle_formula(f, args:)
       f.bottle_specification.rebuild
     else
       ohai "Determining #{f.full_name} bottle rebuild..."
-      versions = FormulaVersions.new(f)
-      rebuilds = versions.bottle_version_map("origin/master")[f.pkg_version]
-      rebuilds.pop if rebuilds.last.to_i.positive?
-      rebuilds.empty? ? 0 : rebuilds.max.to_i + 1
+      FormulaVersions.new(f).formula_at_revision("origin/HEAD") do |prev_f|
+        if f.pkg_version == prev_f.pkg_version
+          prev_f.bottle_specification.rebuild + 1
+        else
+          if f.pkg_version < prev_f.pkg_version
+            opoo <<~EOS
+              Current package version for #{f.full_name} (`#{f.pkg_version}') is lower than version at
+              origin/HEAD (`#{prev_f.pkg_version}').
+
+              Defaulting to no rebuild number, but you may wish to verify this.
+            EOS
+          end
```

Comment:
I'm not sure this is really necessary. It is possible for `f.pkg_version` to be less than `prev_f.pkg_version`, but it does accompany a `version_scheme` bump. Since we already check for decreasing versions in `brew audit`, though, I don't really think a message is needed here.

Reply:
Ah, sure enough, makes sense! Removed...